src/useUrlQueryState.ts

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import { integerFromString, integerToString } from '@kibalabs/core';
 
-export const useUrlQueryState = (name: string, overrideInitialValue?: string | null): [string | null | undefined, (newValue: string | null | undefined) => void] => {
```

Comment:
ah we should keep null for overrideInitialValue - its equivalent to saying delete whatever is there are the start.